app-conf/geonetwork/shutdown.sh

```diff
@@ -1,4 +1,25 @@
 #!/usr/bin/env bash
+#############################################################################
+#
+# Purpose: This script is used to stop geonetwork
+#
+#############################################################################
+
+# Copyright (c) 2009-2018 Open Source Geospatial Foundation (OSGeo)
```

Comment:
We need to check if this is actually a GeoNetwork script and assign the correct license

Reply:
It seems GLP version two
https://geonetwork-opensource.org/manuals/3.2.2/fra/users/html/overview/license.html

Reply:
@sppigot @geonetwork could you please review the license of startup.sh and shutdown.sh
We didn't find this file in @geonetwork source code.

Reply:
@sppigot @Delawen @etj @jesseeichar @ticheler can you please review the license of startup.sh and shutdown.sh included in OSGeoLive? Since those files are not in GeoNetwork source code, can we use LGPL for them?